lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -20,14 +20,32 @@ const Audit = require('../audit');
 const Formatter = require('../../report/formatter');
 
 const KB_IN_BYTES = 1024;
-const WASTEFUL_THRESHOLD_IN_BYTES = 20 * KB_IN_BYTES;
+
+const WASTED_MS_FOR_AVERAGE = 300;
+const WASTED_MS_FOR_POOR = 750;
 
 /**
  * @overview Used as the base for all byte efficiency audits. Computes total bytes
  *    and estimated time saved. Subclass and override `audit_` to return results.
  */
 class UnusedBytes extends Audit {
   /**
+   * @param {number} wastedMs
+   * @return {number}
+   */
+  static scoreForWastedMs(wastedMs) {
```

Comment:
felt cumbersome to use the standard curve here since the points at which we'll style as failure and average are what we really care about, not the median/diminishing points